dali/python/nvidia/dali/plugin/numba/experimental/__init__.py

```diff
@@ -25,6 +25,7 @@
 
 
 _to_numpy = {
+    dali_types.BOOL: "bool_",
```

Comment:
Both `numpy.bool` and `numpy.bool_` are deprecated and removed in newer NumPy. Use plain `bool` instead. This calls for a bit larger rework.